src/orion/core/worker/trial.py

```diff
@@ -233,14 +234,13 @@ def gradient(self):
         """
         return self._fetch_one_result_of_type('gradient')
 
-    @property
-    def is_registered(self):
-        """Check whether `Trial` is registered in database based on `_id` value."""
-        return self._id is not self.NoID
+    def _repr_values(self, values, sep=','):
+        """Represent with a string the given values."""
+        return sep.join(map(lambda value: "{0.name}:{0.value}".format(value), values))
```

Comment:
Isn't pylint nagging about this being a static method? Could be located in `Trial.Value` as one.

Reply:
No, Pylint is surprisingly fine with it

Reply:
Indeed!